wagtail/contrib/search_promotions/views/reports.py

```diff
@@ -31,7 +31,11 @@ class SearchTermsReportView(ReportView):
     index_results_url_name = "wagtailsearchpromotions:search_terms_results"
     columns = [
         Column("query_string", label=_("Search term(s)"), sort_key="query_string"),
-        Column("_hits", label=_("Views"), sort_key="_hits"),
+        NumberColumn(
+            "_hits",
+            label=_("Views"),
+            sort_key="_hits",
+        ),
```

Comment:
To keep the diff minimal

```suggestion
        NumberColumn("_hits", label=_("Views"), sort_key="_hits"),
```